popsup.js

```diff
@@ -0,0 +1,181 @@
+    technologiesMobile: ['Ruby', 'CSS', 'JavaScript'],
+    technologiesDesktop: [
+      'CodeKit',
+      'GitHub',
+      'JavaScript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    sourceLink: 'https://github.com/Maxwell011',
+    liveLink: 'https://github.com/Maxwell011/My-Portfolio.git',
+  },
+  {
+    id: 6,
+    name: 'Keeping track of hundreds of components',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    imageDesktop: './images/Snapshoot desktop.png',
+    githubIcon: './images/github.png',
+    liveIcon: './images/Icon - Export.png',
+    technologiesMobile: ['Ruby', 'CSS', 'JavaScript'],
+    technologiesDesktop: [
+      'CodeKit',
+      'GitHub',
+      'JavaScript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    sourceLink: 'https://github.com/Maxwell011',
+    liveLink: 'https://github.com/Maxwell011/My-Portfolio.git',
+  },
+];
+
+const myModal = document.getElementById('myModal');
+// Get the <span> element that closes the modal
+
+/* eslint-disable no-unused-vars */
+const closeModal = () => {
+  myModal.style.display = 'none';
+};
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function modal(event) {
+  if (event.target === myModal) {
+    myModal.style.display = 'none';
+  }
+};
+/* eslint-disable no-unused-vars */
+function viewProjectDetails(projectIndex) {
+  const project = data[projectIndex];
+  myModal.innerHTML = `<div class="modal-content">
+<div class="modal-header">
+  <div class="bg-img">
+  <span onclick="closeModal()" id="closeBtn" class="close">&times;</span>
+  <img class="desktop-bg" src="${project.imageDesktop}" />
+  </div>
+</div>
+<div class="modal-body">
+  <div class="title-container">
+    <h3 id="title">${project.name}</h3>
+  </div>
+  <ul class="tags" id="tech">
+  ${project.technologiesDesktop.map((ele) => `<li>${ele}</>`)}
+  </ul>
+  <ul class="tags" id="mobile-ul">
+  ${project.technologiesMobile.map((ele) => `<li>${ele}</>`)}
+  </ul>
+  <p id="description">${project.description}</p>
+  <div class="modal-footer">
+  <a href="${project.liveLink}">
+    <button type="submit" class="project-view-btn">
+      <span> See Live</span>
+      <img
+        src="${project.liveIcon}"
+        alt="See Live Project"
+      /></button
+  ></a>
+  <a href="${project.sourceLink}">
+    <button type="submit" class="project-view-btn">
+      <span>See Source</span>
+      <img
+        src="${project.githubIcon}"
+        alt="See Source Code"
+      /></button
+  ></a>
```

Comment:
- [ ] Please remove the black border from the buttons and make sure the button's texts have equal padding from all sides.

### Your design
![image](https://user-images.githubusercontent.com/83803414/189148033-1d7bca37-4b86-4370-ad6b-2763092833cd.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/189148076-ebc74355-142b-49e2-8739-af2b7e8bae6a.png)